### PR TITLE
fix: override `Array.prototype.map`

### PR DESCRIPTION
**Pull request by SlayerOrnstein**
### What did you fix? 
Same fix as #566 but @Sainan is unable to rebase due to #562 

---

### Reproduction steps

---

### Considerations
- Does this contain a new dependency? **No**
- Does this introduce opinionated data formatting or manual data entry? **No**
- Does this pr include updated data files in a separate commit that can be reverted for a clean code-only pr? **No**
- Have I run the linter? **Yes**
- Is is a bug fix, feature request, or enhancement? **Bug Fix**

### CODE REVIEW

**Comment by Sainan** [comment]
Just FYI, you can merge a PR even if it's not rebased.

**Comment by TobiTenno** [comment]
we're aware, but you were missing half the implementation

**Comment by Sainan** [comment]
I'm not sure what you mean, I tested locally with `npm install ../warframe-items` and the map function was working as expected.

**Comment by Sainan** [comment]
Nevermind, I see it was also added to the .mjs file here...

So much overengineering here yet still having code duplication like that? Wow.

**Comment by Sainan** [comment]
I think I was being quite respectful and professional.

Now I am just mocking you because you clearly had a very big ego about how overengineered this package is to the point were you actually banned when I suggested it should be simpler so it would work on Windows without problems, for example. 😀

**Comment by SlayerOrnstein** [comment]
So completely unaware, got it. So I'm just gonna say this, if you're going to contribute follow the rules set by the repo not "you can merge without rebasing" if you don't want to that's fine you also don't have to contribute 

That is the end of this whole conversion, you were told what needed to be told and you refused, anything after this as harsh as it seems falls under harassment

**Comment by Sainan** [comment]
Harass deez

**Comment by Sainan** [comment]
Like, how the fuck can you equate me reporting issues to "harassment"? You people should not be doing software development if your ego and emotions are this unstable.

File: build/parser.mjs [hunk 1]
```diff
@@ -532,6 +532,7 @@ class Parser {
         .replace('/', '')
         .replace(/[ /*]/g, '-')
         .replace(/[:<>[\]?!]/g, '')
+        .replaceAll('"','')
```
Comment (TobiTenno) [inline build/parser.mjs:535]:
```suggestion
        .replace(/[:<>[\]?!"]/g, '')
```

**Review by TobiTenno — APPROVED** [review]

**Pull request merged**